File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/CatalogEntity.java
package com.tokopedia.tokopoints.view.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class CatalogEntity {
    @Expose
    @SerializedName("catalogs")
    private ArrayList<CatalogsValueEntity> catalogs;

    @SerializedName("pageInfo")
    private TokopointPaging paging;

    public ArrayList<CatalogsValueEntity> getCatalogs() {
        return catalogs;
    }

    public void setCatalogs(ArrayList<CatalogsValueEntity> catalogs) {
        this.catalogs = catalogs;
    }

    public TokopointPaging getPaging() {
        return paging;
    }

    public void setPaging(TokopointPaging paging) {
        this.paging = paging;
    }

    @Override
    public String toString() {
        return "CatalogEntity{" +
                "catalogs=" + catalogs +
                '}';
    }
}

File: features/promo/tokopoints/src/main/java/com/tokopedia/tokopoints/view/model/CatalogEntity.kt
package com.tokopedia.tokopoints.view.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.tokopedia.library.baseadapter.BaseItem
import com.tokopedia.tokopoints.view.model.section.CountDownInfo
import java.io.Serializable
import java.util.ArrayList

data class CatalogEntity(

        @Expose
        @SerializedName("catalogs")
        var catalogs: ArrayList<CatalogsValueEntity>? = null,

        @SerializedName("pageInfo")
        var paging: TokopointPaging? = null,

        @SerializedName("countdownInfo")
        val countDownInfo: CountDownInfo? = null
)